addons/mail/models/mail_message.py

```diff
@@ -1032,6 +1023,23 @@ def _message_format(self, format_reply=True, msg_vals=None):
                 vals["thread"] = thread
         return vals_list
 
+    def _message_format_perso(self, format_reply=True, msg_vals=None):
```

Comment:
Ok, makes sense then with some context. Would avoid the "perso" suffix which is a bit weird (``internals`` ? ``user_information`` ?). However why isn't it a parameter on ``_message_format(self, ..., include_internals=False)``? Because you might end up with people overriding the wrong method, not calling the right method, ... Seems it is more an option of the base method than a really new method, especially they are private and do not really risk leaking information by themselves.

Reply:
Well, I couldn't decide between a method and a param so made the method.
But I think a param would be better now, Having too many format methods isn't a good thing.